WKData/Sources/WKData/Extensions/Dictionary+API.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+extension Dictionary where Key == String, Value == Any {
+    
+    func encodedForAPI() -> [String: Any] {
+        var returnDict: [String: Any] = [:]
+        
+        for (key, value) in self {
+            
+            guard
+                let encodedName = key.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlQueryComponentAllowed),
+                let encodedValue = (value as? String)?.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlQueryComponentAllowed) else {
+                    continue
```

Comment:
Unsure if we want to continue our pattern of `assert(...)` in the new frameworks, but this may be a good candidate. Any key/value parameter pair failing to be properly URL encoded means the entire parameter dictionary is invalid, right?

Reply:
Fixed!